server/src/api/commits.ts

```diff
@@ -16,10 +16,34 @@
 
 /**
  * @fileoverview Handles routing of /commits endpoints.
- * @author Karen Frilya Celine
```

Comment:
You probably don't want to delete the original author. Instead, add your own name into the author list.

Reply:
@xlai20 Oh I'm just removing this to standardise the files because if you check all other files, you will notice that we don't have the author tag anywhere else. This was added when Karen was creating the empty files during the setup.